.github/actions/setup/action.yml

```diff
@@ -6,7 +6,7 @@ runs:
 
     - uses: actions/setup-node@v3
       with:
-        node-version: 16
+        node-version: 18.18.0
```

Comment:
Why did we not have to change this before? Why is it that we need to change this when we merge to master?

Reply:
Updated